services/storage/app/models/upload_storage.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class UploadStorage < Storage::ApplicationRecord
+  belongs_to :tenant
```

Comment:
you can remove the `belongs_to tenant` and tenant `has_one`. We are using DB schemas to isolate the tenants so all records automatically belong to a specific tenant